third_party/workspace.bzl

```diff
@@ -65,20 +65,21 @@ def tensorboard_workspace():
       generated_rule_name = "archive",
   )
 
-  # Roughly corresponds to Chrome 55
+  # Last Updated: 2018-11-29
+  # Roughly corresponds to Chrome 70
   filegroup_external(
       name = "org_chromium_chromium",
       licenses = ["restricted"],  # So many licenses
       sha256_urls = {
-          "e3c99954d6acce013174053534b72f47f67f18a0d75f79c794daaa8dd2ae8aaf": [
-              "https://mirror.bazel.build/commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/423768/chrome-linux.zip",
-              "http://commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/423768/chrome-linux.zip",
+          "ac11a4b8c901622cdd6cebbdac7a0dfe20c32f4d287a44ed19368444a2cd159e": [
+              "https://mirror.bazel.build/commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/582301/chrome-linux.zip",
```

Comment:
This URL appears to be broken?

```
$ curl --silent https://mirror.bazel.build/commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/582301/chrome-linux.zip 
<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: bazel-mirror/commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/582301/chrome-linux.zip</Details></Error>
```

The corresponding URL prior to this change works.

Reply:
Yup, thats expected. Bazel mirror is something that I am not too familiar with but I believe it has to be populated explicitly (which I think is terrible for maintainability). Btw, if the first link fails, it resorts to subsequent urls so that should be fine.

(Also, I need to make sure mirror gets populated after talking to someone :()